app/controllers/concerns/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+
+  def index
+
+  end
```

Comment:
Use single line syntax if the action is empty.
```suggestion
  def index; end
```